.eslintrc.js

```diff
@@ -64,8 +69,18 @@ module.exports = {
         'no-unused-expressions': 'off',
         'mocha/no-exclusive-tests': 'error',
         'mocha/no-mocha-arrows': 'error',
```

Comment:
While I'm asking the question about `sort-keys`, hating the `mocha-no-arrows` ;)

Reply:
@solidgoldpig There is a reason this is there...[mocha discourages](https://mochajs.org/#arrow-functions) the use of arrow functions ;)

Reply:
I understand why - just saying I hates it ;)  Fortunately the auto-fixing sorts it, but I've developed a weird memory muscle now to type `function(){}`

Reply:
You use VS code too right? I've found the [mocha snippets extension](https://github.com/spoonscen/es6-mocha-snippets-vs-code) a godsend for speeding up writing test blocks :)

You'll need to prefix the shortcuts with `f` to use the non-arrow functions:

`fdes` + <kbd>tab</kbd> = 🥳

Reply:
Giving it a go

Reply:
```json
{
    "mocha-snippets.function-type": 'arrow' | 'function' | 'both'
}
```